src/Codeception/Module/Symfony.php

```diff
@@ -22,6 +22,10 @@
  *
  * * Maintainer: **raistlin**
  * * Stability: **stable**
+ * 
+ * ## Compatibility
+ *
+ * This module is compatible with all other Codeception modules, except WebDriver and the modules for other PHP frameworks.
```

Comment:
`4.` @DavertMik should know, because he implemented it.
`7.` no, features are not tied to specific suite names.
`8.` these parameters aren't required, because they have default values.
`9.` part names are module specific, please refer to Parts section of documentation.
`10.` no.